packages/@glimmer/compiler/lib/passes/1-normalization/keywords/append.ts

```diff
@@ -185,4 +185,53 @@ export const APPEND_KEYWORDS = keywords('Append')
       });
       return Ok(new mir.AppendTextNode({ loc: node.loc, text }));
     },
+  })
+  .kw('component', {
```

Comment:
Can you call this change out a bit more explicitly in the description? Specifically, this _probably_ means that:

```hbs
{{component}}
```

Where `{{component}}` is ultimately going to be using `{{this.component}}`. (I mean, its bonkers, not saying we have to support it, but I want to call it out as I'm **certain** _someone_ has done ^).